app/presenters/tasks/ioj.rb

```diff
@@ -18,8 +18,16 @@ def in_progress?
     end
 
     def completed?
-      crime_application.ioj_passport.any? ||
+      return true if ioj_passported?
+
+      crime_application.ioj.present? &&
```

Comment:
Was wondering if it might be worth putting the:
```ruby
def ioj
  @ioj ||= crime_application.ioj
end
```
method in the base presenter?

Reply:
In this instance, the base presenter is for the task list, and each task class affects mostly one record (applicant, case, ioj, etc.) so I think is best not to raise these to the base class and that way everything is more self-contained and easy to read the code.

Perhaps if these task classes become more complex then definitely it might be worth considering shifting some code to the base task class.